bot/exts/moderation/watchchannels/_watchchannel.py

```diff
@@ -285,7 +285,10 @@ async def send_header(self, msg: Message) -> None:
         else:
             message_jump = f"in [#{msg.channel.name}]({msg.jump_url})"
 
-        footer = f"Added {time_delta} by {actor} | Reason: {reason}"
+        # Add reason to the footer if it exists.
+        footer = f"Added {time_delta} by {actor}"
+        if reason:
+            footer += f" | Reason: {reason}"
```

Comment:
This should be put back to the way it was before, since there will always be a reason now.
```suggestion
        footer = f"Added {time_delta} by {actor} | Reason: {reason}"
```